src/Authentication/Authentication.Core/Utilities/AuthenticationHelpers.cs

```diff
@@ -133,7 +133,11 @@ private static async Task<InteractiveBrowserCredential> GetInteractiveBrowserCre
                 }
                 else
                 {
-                    authRecord = await interactiveBrowserCredential.AuthenticateAsync(new TokenRequestContext(authContext.Scopes), cancellationToken).ConfigureAwait(false);
+                    authRecord = await Task.Run(() =>
+                    {
+                        // Run the thread in MTA.
+                        return interactiveBrowserCredential.AuthenticateAsync(new TokenRequestContext(authContext.Scopes), cancellationToken);
```

Comment:
Added to unblock auth popup dialog not opening due to changes made in Azure.Identity 1.10.x - https://github.com/Azure/azure-sdk-for-net/pull/36061.